tests/seeyou/test_reader.py

```diff
@@ -239,6 +239,13 @@ def test_original():
             check_waypoint(waypoint)
 
 
+def test_read():
+    with open(SIMPLE_CUPFILE) as f:
+        waypoints = Reader().read(f)['waypoints']
+        for index in range(len(waypoints)):
```

Comment:
this should probably use https://docs.python.org/2/library/functions.html#enumerate